creator-node/src/apiHelpers.ts

```diff
@@ -135,14 +137,38 @@ export const sendResponse = (
   res.status(resp.statusCode).send(resp.object)
 }
 
+/**
+ * Wrapper helper method to send response with metrics involved
+ * @param {Object} req Express request object
+ * @param {Object} res Express response object
+ * @param {Object} routeResponse error or success response object
+ * @param {Object} prometheusResult prometheus results to track
+ */
+export const sendResponseWithMetric = (
+  req: CustomRequest,
+  res: Response,
+  routeResponse: ApiResponse,
+  prometheusResult: { result: string; mode?: string }
+) => {
+  if (req.endMetricTimer) {
+    try {
+      req.endMetricTimer(prometheusResult)
+    } catch (e: any) {
+      req.logger.warn(`Could not end timer: ${e.message}`)
+    }
+  }
+
+  sendResponse(req, res, routeResponse)
```

Comment:
prob better to `return sendResponse()` here - no issue now since that fn doesn't return anything but good for consistency